chromium_src/components/omnibox/browser/autocomplete_provider_client.cc

```diff
@@ -8,3 +8,8 @@
 std::u16string AutocompleteProviderClient::GetClipboardText() const {
   return u"";
 }
+
+bool AutocompleteProviderClient::IsQuerySafeToSearchSuggestions(
+    const std::u16string& query) const {
+  return true;
```

Comment:
isn't this going to skip the filter for other search engines like google, etc...?

Reply:
FWIW, I misread it in the same way.